tests/jballerina-unit-test/src/test/resources/test-src/types/table/xml-type-table-key.bal

```diff
@@ -83,14 +83,16 @@ function testMemberAccessWithInvalidXMLRecordKey() {
     table<Customer> key(id) customerTable = table [{id: xml `<id>123</id>`, name: {fname: "Sanjiva", lname: "Weerawarana"}, address: "Sri Lanka" },
                                                    {id: xml `<id>234</id>`, name: {fname: "James" , lname: "Clark"}, address: "Thailand" }];
 
-    Customer customer = customerTable[xml `<id>245</id>`];
+    Customer? customer = customerTable[xml `<id>245</id>`];
+    assertEquality((), customer);
 }
 
 function testMemberAccessWithInvalidXMLMultiKey() {
     table<Customer> key(id, name) customerTable = table [{id: xml `<id>123</id>`, name: {fname: "Sanjiva", lname: "Weerawarana"}, address: "Sri Lanka" },
                                                    {id: xml `<id>234</id>`, name: {fname: "James" , lname: "Clark"}, address: "Thailand" }];
 
-    Customer customer = customerTable[xml `<id>245</id>`, {fname: "Sanjiva", lname: "Weerawarana"}];
```

Comment:
This should have (and should) result in a compilation error, right? The static type of the member access here should be `Customer?`. 

>  If T is a subtype of table<R>, then the static type of member-access-expr is R?.

I don't think we've fixed that yet. Shall we fix it and add negative tests?

Reply:
Created an issue for this too - https://github.com/ballerina-platform/ballerina-lang/issues/28474.

Reply:
Fixed in dffc78b